code/game/machinery/doors/door.dm

```diff
@@ -264,10 +264,7 @@
 	if(!user.combat_mode && istype(I, /obj/item/fireaxe))
 		try_to_crowbar(I, user, FALSE)
 		return TRUE
-	else if(I.item_flags & NOBLUDGEON || user.combat_mode)
-		return FALSE
-
-	else if(try_to_activate_door(user))
+	else if((!(I.item_flags & NOBLUDGEON) && !user.combat_mode) && try_to_activate_door(user))
 		return TRUE
 	return ..()
```

Comment:
I'm not a huge fan of stacking all these checks in one block. Couldn't you just return ..() instead of false in the old check?